test/test_gdrcopy.py

```diff
@@ -23,6 +23,7 @@
 import unittest
 
 from helpers import centos, docker, ubuntu
+from distutils.version import LooseVersion
```

Comment:
This should go in the building block, not the test.

Reply:
Thanks @samcmill . Is there a way I can run the tests without pushing new changes to you each time?

Reply:
Nevermind, I see it's running now!